__init__.py

```diff
@@ -1720,7 +1730,7 @@ def _send_batch(self, base_url, endpoint, batch, dataset_id=None, dataset_versio
 
         """
         try:
-            params = {'data': base64.b64encode(json.dumps(batch))}
+            params = {'data': base64.b64encode(json.dumps(batch).encode('utf-8'))}
```

Comment:
don't you have to like `.decode` it as well or something iirc? See this thread when I was helping Cat w/ py3 https://mixpanel.slack.com/archives/C03QFM9BL/p1614279849203600?thread_ts=1614278061.196100&cid=C03QFM9BL

Reply:
fixed in later a commit on the full PR